sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -151,6 +164,8 @@ private[sbtplugin] object ScalaJSPluginInternal {
       scalaJSLinkerConfigFingerprint in key :=
         StandardConfig.fingerprint((scalaJSLinkerConfig in key).value),
 
+      moduleName in legacyKey := (moduleName in key).value,
```

Comment:
This one is arguably OK because it will still affect the `artifactPath in legacyKey`. Nevertheless, I think we should also derive the new one from the old one, for consistency.